Migrations/ChefsterDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,138 @@
+﻿// <auto-generated />
```

Comment:
Can we auto generate these easily in the future if we ever update the DB? This is super sweet btw

Reply:
I believe so. from my understanding, entity framework creates these for us whenever we do migrations